wagtail/images/models.py

```diff
@@ -651,17 +651,22 @@ def create_renditions(
         return_value: Dict[Filter, AbstractRendition] = {}
         filter_map: Dict[str, Filter] = {f.spec: f for f in filters}
 
-        with self.open_file() as file:
-            original_image_bytes = file.read()
+        _, tempfile = mkstemp()
```

Comment:
We import `gettext_lazy` as `_` - https://github.com/wagtail/wagtail/blob/927f7d693bf4316dbdf918272ed1f6a8b4b45e3a/wagtail/images/models.py#L30